packages/appstore/src/components/add-more-wallets/carousel-container.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import useEmblaCarousel, { EmblaCarouselType, EmblaOptionsType } from 'embla-carousel-react';
+import { PrevButton, NextButton } from './carousel-buttons';
+import { observer, useStore } from '@deriv/stores';
+
+const CarouselContainer: React.FC<React.PropsWithChildren<unknown>> = observer(({ children }) => {
+    const { ui } = useStore();
+    const { is_mobile } = ui;
+
+    const options: EmblaOptionsType = {
+        align: 0,
+        containScroll: 'trimSnaps',
+        watchDrag: !!is_mobile,
```

Comment:
I think `is_mobile` is `boolean` and you don't need to use `!!` to convert it to `boolean`.